schema/schema.go

```diff
@@ -6,161 +6,161 @@ type locationSchema struct {
-	Name          string `jsonschema:"name,description:e.g. XYZ Certified Application Specialist (MCAS) - [Add the certificate name]"`
-	Website       string `jsonschema:"website,description:Link to issuing authority's description of the certificate,format:uri"`
-	Verification  string `jsonschema:"verification,description:External candidate verification URL,format:uri"`
-	GrantDate     string `jsonschema:"grantDate,description:e.g. 2017-06-29 - [resume.json uses the ISO 8601 date standard],format:date"`
-	Score         string `jsonschema:"score,description:e.g. 95% - [Exam result (PASS/FAIL 100% 100)]"`
-	EndDate       string `jsonschema:"endDate,description:e.g. 2017-06-29 - [resume.json uses the ISO 8601 date standard],format:date"`
-	DoesNotExpire bool   `jsonschema:"doesNotExpire,format:checkbox"`
+	Code          string `jsonschema:"code;description:e.g. 1Z0-062"`
+	Name          string `jsonschema:"name;description:e.g. XYZ Certified Application Specialist (MCAS) - [Add the certificate name]"`
+	Website       string `jsonschema:"website;description:Link to issuing authority's description of the certificate;format:uri"`
+	Verification  string `jsonschema:"verification;description:External candidate verification URL;format:uri"`
+	GrantDate     string `jsonschema:"grantDate;description:e.g. 2017-06-29 - [resume.json uses the ISO 8601 date standard];format:date"`
+	Score         string `jsonschema:"score;description:e.g. 95% - [Exam result (PASS/FAIL, 100%, 100)]"`
+	EndDate       string `jsonschema:"endDate;description:e.g. 2017-06-29 - [resume.json uses the ISO 8601 date standard];format:date"`
+	DoesNotExpire bool   `jsonschema:"doesNotExpire;format:checkbox"`
 }
 
 type referenceSchema struct {
-	Name      string `jsonschema:"name,description:e.g. Stephan Mark"`
-	Company   string `jsonschema:"company,description:e.g. Xyz"`
-	Position  string `jsonschema:"position,description:e.g. Senior Software Engineer"`
-	Reference string `jsonschema:"reference,description:e.g. Joe blogs was a great employee who turned up to work at least once a week. He exceeded my expectations when it came to doing nothing."`
+	Name      string `jsonschema:"name;description:e.g. Stephan Mark"`
+	Company   string `jsonschema:"company;description:e.g. Xyz"`
+	Position  string `jsonschema:"position;description:e.g. Senior Software Engineer"`
+	Reference string `jsonschema:"reference;description:e.g. Joe blogs was a great employee, who turned up to work at least once a week. He exceeded my expectations when it came to doing nothing."`
 }
 
 type languageSchema struct {
-	Language string  `jsonschema:"language,description:e.g. English - [Name of language]"`
-	Score    float64 `jsonschema:"score,description:e.g. 20 - [Score for the language]"`
+	Language string  `jsonschema:"language;description:e.g. English - [Name of language]"`
+	Score    float64 `jsonschema:"score;description:e.g. 20 - [Score for the language]"`
```

Comment:
Maybe reuse the `Score` struct? That way one could add the type of a score.

Reply:
Fixed in 39d8490